ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -1703,6 +1703,126 @@ class EclCmdStatus : public EclCmdCommon
     unsigned int       optListLimit;
 };
 
+class EclCmdZapGen : public EclCmdCommon
+{
+public:
+    EclCmdZapGen()
+    {
+        optObj.accept = eclObjWuid;
+    }
+    virtual eclCmdOptionMatchIndicator parseCommandLineOptions(ArgvIterator &iter)
+    {
+        eclCmdOptionMatchIndicator retVal = EclCmdOptionNoMatch;
+        if (iter.done())
+            return EclCmdOptionNoMatch;
+
+        for (; !iter.done(); iter.next())
+        {
+            const char *arg = iter.query();
+            if (iter.matchOption(optWuid, ECLOPT_WUID)||iter.matchOption(optWuid, ECLOPT_WUID_S))
+            {
+                optObj.type = eclObjWuid;
+                retVal = EclCmdOptionMatch;
+                continue;
+            }
+            if (iter.matchOption(optPath, ECLOPT_PATH))
+            {
+                retVal = EclCmdOptionMatch;
+                continue;
+            }
+            if (iter.matchOption(optIncThorSlave, ECLOPT_INC_THOR_SLAVE_LOGS))
```

Comment:
Should use iter.matchFlag() for boolean options.  then you also don't need "--include-slave-logs=1".  To turn on just add "--include-slave-logs", default is off.

Reply:
Changed